app/views/alaveteli_pro/comment/_suggestions.html.erb

```diff
@@ -6,7 +6,7 @@
 <ul>
   <% if [ 'successful', 'partially_successful' ].include?(@info_request.described_state) %>
     <li><%= _(' <strong>Summarising</strong> the content of any information returned. ')%></li>
-    <li><%= _(" Say how you\'ve <strong>used the information</strong>, with " \
+    <li><%= _(" Say how you've <strong>used the information</strong>, with " \
```

Comment:
Since we're modifying these strings, might as well remove the empty space at the start.

Might as well also do so for  `' <strong>Summarising</strong> the content of any information returned. '` too.

Applies to both files in this diff.

Reply:
This change is actually to retain the same string as before. Without this change backward slash is now escaped. so the string appears different in the POT/PO files.

I beleive this could be due to same escaping bug which effected the new lines.

Reply:
Ah okay gotcha. In that case yeah lets just leave it as it is for now